src/cloud/schemas/ProvisionUserDeleteRequest.yml

```diff
@@ -0,0 +1,26 @@
+type: object
+properties:
+  userID:
+    type: string
+    description: user id to delete
+  orgID:
+    type: string
+    description: organization id the user should be deleted from
+  newOwner:
```

Comment:
Again `newOwnerID`

Reply:
Good idea - updated! I'll update it on the IDPE end as well.